resources/leiningen/new/kraken_http_api/project.clj

```diff
@@ -15,7 +15,7 @@
                  ;; kehaar.core/go-handler. (This may not be true anymore in
                  ;; core.async 0.2.x; need to test.)
                  [org.clojure/core.async "0.2.374"]
-                 [democracyworks/kehaar "0.5.0"]
+                 [democracyworks/kehaar "0.8.1"]
 
                  [io.pedestal/pedestal.service "0.4.1"]
                  [io.pedestal/pedestal.service-tools "0.4.1"]
```

Comment:
Can we update the rest of the deps (that don't require other changes) while we're in here? Fine if you want to do it in another PR.